src/stt/src/SteinerTreeBuilder.cpp

```diff
@@ -116,6 +116,49 @@ float SteinerTreeBuilder::getAlpha(const odb::dbNet* net) const
   return net_alpha;
 }
 
+// This checks whether the tree has the property that no two
+// non-adjacent edges have intersecting bounding boxes.  If
+// they do it is a failure as embedding those egdes may cause
+// overlap between them.
+bool SteinerTreeBuilder::checkTree(const Tree& tree) const
+{
+  // Such high fanout nets are going to get buffered so
+  // we don't need to worry about them.
+  if (tree.deg > 100) {
```

Comment:
Do you suspect that there is any performance left on the table by defaulting to a value of 100? Is 80 also a valid value? Is there a way to calculate a dynamic value instead?

Reply:
It is a magic number and others might work as well.   I don't observe much change in results when running public & secure CI so I don't think it matters too much.  This is mostly to fix up the case of invalid results that cause grt to fail with the 'index not found error'.  

In nangate45/swerv I generated some stats and about 8% of nets used the fallback (that was before the alpha-0.1 fallback so it might be less now).

Reply:
@jjcherry56 is also working on some PD fixes that might lower the frequency as well.

Reply:
The current algorithm is n^2 so the limit is also to help with runtime.  I considered writing a scanline version of this to get n*logN and removing the limit.  However I don't think it is worth the effort, as I mentioned those nets would likely get buffered before ever reaching grt and rsz doesn't really care (it really only needs a MST).